examples/index.html

```diff
@@ -19,37 +19,37 @@
         (click on title to open fullscreen)
         </p>
         <div class="exampleContainer">
-            <h3><a href='./orthographic.html'>Orthographic</a></h3>
+            <h3><a href='./orthographic.html' target='_blank'>Orthographic</a></h3>
```

Comment:
Not really important, but I prefer the old behaviour (without `target='_blank'`) as it allows both behaviour (normal click, and middle click or ctrl+click). We can let user choose :-)